clojure-mode.el

```diff
@@ -855,19 +855,19 @@ any number of matches of `clojure--sym-forbidden-rest-chars'."))
       ;; Java interop highlighting
       ;; CONST SOME_CONST (optionally prefixed by /)
       ("\\(?:\\<\\|/\\)\\([A-Z]+\\|\\([A-Z]+_[A-Z1-9_]+\\)\\)\\>" 1 font-lock-constant-face)
+      ;; special handling for foo.bar.baz in the namespace definition
```

Comment:
I'm a bit confused by this comment. What exactly is special here?

Reply:
Good question. It might be better to either: (A) write here:
```
      ;; special handling for namespace definition
```
or (B) remove the comment in the first place.

And BTW I have a feeling the lines 865 to 870:
```
        ;; Foo Bar$Baz Qux_ World_OpenUDP Foo. Babylon15.
        ("\\(?:\\<\\|\\.\\|/\\|#?^\\)\\([A-Z][a-zA-Z0-9_]*[a-zA-Z0-9$_]+\\.?\\>\\)" 1 font-lock-type-face)
        ;; foo.bar.baz
        ("\\<^?\\([a-z][a-z0-9_-]+\\.\\([a-z][a-z0-9_-]*\\.?\\)+\\)" 1 font-lock-type-face)
```
are kind of a redundant here but it would be a too much of a change in one PR (at least for me) so I rather leave it untouched.

Reply:
Yeah, that's probably true. I think there are some tests for them, so if you remove them you'd know pretty soon. :-)